fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/util/SumoRouteFileCreator.java

```diff
@@ -64,138 +59,140 @@
 
```

Comment:
Update class header of SumoRouteFileCreator

Reply:
The javadoc of the class states "This class is used to write Sumo vehicle definitions and routes ..", but it doesn't write Routes anymore

Reply:
Ah okay, I was confused by "class header" 😅 
Will do!

Reply:
Maybe we should consider renaming the class as well?

Reply:
hm.. from SUMOs perspective this is still a "Route file"

Reply:
alright. I will leave it as is.